editor/src/clj/editor/debug_view.clj

```diff
@@ -484,9 +484,9 @@
        :content (.getMessage exception)})))
 
 (defn start-debugger!
-  [debug-view project target-address]
+  [debug-view project target-address instance-index]
   (try
-    (mobdebug/connect! target-address mobdebug-port
+    (mobdebug/connect! target-address (+ mobdebug-port instance-index)
```

Comment:
Use a separate debugger port for each instance to be able to debug them all (not simultaneously, no, picking one in Target menu)